test/CMakeLists.txt

```diff
@@ -362,6 +362,17 @@ if (${CMAKE_SYSTEM_NAME} MATCHES "Linux" AND CMAKE_CXX_COMPILER_ID STREQUAL "GNU
     )
 endif()
 
+# Only produce timing tests for UNIX based systems (faketime constraint)
+if (NOT WIN32)
+    add_test(
+            NAME "tiledb_timing_unit"
+            COMMAND tiledb_unit --durations=yes [sub-millisecond]
+            WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+    )
+
+    set_property(TEST tiledb_timing_unit PROPERTY ENVIRONMENT_MODIFICATION "FAKETIME=set:2020-12-24 20:30:00;LD_PRELOAD=set:${CMAKE_BINARY_DIR}/vcpkg_installed/x64-linux/lib/liblibfaketime.so")
```

Comment:
You should call `find_library` to get the path. And instead of if not on Windows, you should write if the library was not found.

Reply:
Done